docs/community.vmware.vmware_guest_module.rst

```diff
@@ -49,7 +49,7 @@ Parameters
                         <span style="color: purple">list</span>
                          / <span style="color: purple">elements=dictionary</span>
                     </div>
-                    <div style="font-style: italic; font-size: small; color: darkgreen">added in 1.7.0</div>
+                    <div style="font-style: italic; font-size: small; color: darkgreen">added in 1.8.0</div>
```

Comment:
I'm surprised by this change and the fact there is not associated changelog entry.

Reply:
The original PR #1156 didn't have have a changelog fragment, either.

I've approved the original PR without a changelog fragment because I thought it unnecessary.  I thought we could fix a wrong `version_added` just like this, without adding a changelog fragment.

I thought a documentation fix like this doesn't need a changelog fragment. If you think it does, please let me know.

Reply:
@goneri #179 was opened before, but merged after the 1.7.0 release. Do you want me to add a changelog fragment for this documentation fix?

Reply:
Ok, thanks for the feedback.